packages/tests/tests/relay/interop.node.spec.ts

```diff
@@ -0,0 +1,168 @@
+import { Protocols } from "@waku/interfaces";
+import { createRelayNode } from "@waku/sdk";
+import { bytesToUtf8, utf8ToBytes } from "@waku/utils/bytes";
+import { expect } from "chai";
+
+import {
+  delay,
+  makeLogFileName,
+  NOISE_KEY_1,
+  NOISE_KEY_2,
+  tearDownNodes
+} from "../../src/index.js";
+import { MessageRpcResponse } from "../../src/node/interfaces.js";
+import { base64ToUtf8, NimGoNode } from "../../src/node/node.js";
+
+import { TestContentTopic, TestDecoder, TestEncoder } from "./utils.js";
+
+describe("Waku Relay, Interop", function () {
+  this.timeout(15000);
+  let waku: RelayNode;
+  let nwaku: NimGoNode;
+
+  beforeEach(async function () {
+    this.timeout(30000);
+    waku = await createRelayNode({
+      staticNoiseKey: NOISE_KEY_1
+    });
+    await waku.start();
+
+    nwaku = new NimGoNode(this.test?.ctx?.currentTest?.title + "");
+    await nwaku.start({ relay: true });
+
+    await waku.dial(await nwaku.getMultiaddrWithId());
+    await waitForRemotePeer(waku, [Protocols.Relay]);
+
+    // Nwaku subscribe to the default pubsub topic
+    await nwaku.ensureSubscriptions();
+  });
+
+  afterEach(async function () {
+    this.timeout(15000);
+    await tearDownNodes(nwaku, waku);
+  });
+
+  it("nwaku subscribes", async function () {
+    let subscribers: PeerId[] = [];
+
+    while (subscribers.length === 0) {
+      await delay(200);
+      subscribers =
+        waku.libp2p.services.pubsub!.getSubscribers(DefaultPubSubTopic);
+    }
+
+    const nimPeerId = await nwaku.getPeerId();
+    expect(subscribers.map((p) => p.toString())).to.contain(
+      nimPeerId.toString()
+    );
+  });
+
+  it("Publishes to nwaku", async function () {
+    const messageText = "This is a message";
+    await waku.relay.send(TestEncoder, { payload: utf8ToBytes(messageText) });
+
+    let msgs: MessageRpcResponse[] = [];
+
+    while (msgs.length === 0) {
+      await delay(200);
+      msgs = await nwaku.messages();
+    }
+
+    expect(msgs[0].contentTopic).to.equal(TestContentTopic);
+    expect(msgs[0].version).to.equal(0);
+    expect(base64ToUtf8(msgs[0].payload)).to.equal(messageText);
+  });
+
+  it("Nwaku publishes", async function () {
+    await delay(200);
+
+    const messageText = "Here is another message.";
+
+    const receivedMsgPromise: Promise<DecodedMessage> = new Promise(
+      (resolve) => {
+        void waku.relay.subscribe<DecodedMessage>(TestDecoder, (msg) =>
+          resolve(msg)
+        );
+      }
+    );
+
+    await nwaku.sendMessage(
+      NimGoNode.toMessageRpcQuery({
+        contentTopic: TestContentTopic,
+        payload: utf8ToBytes(messageText)
+      })
+    );
+
+    const receivedMsg = await receivedMsgPromise;
+
+    expect(receivedMsg.contentTopic).to.eq(TestContentTopic);
+    expect(receivedMsg.version!).to.eq(0);
+    expect(bytesToUtf8(receivedMsg.payload!)).to.eq(messageText);
+  });
+
+  describe.skip("Two nodes connected to nwaku", function () {
```

Comment:
Why skip?

Reply:
Thanks, Great catch. I've missed this .skip because It was an existing test that was skipped for some reason.
However I see that It can be fixed and seems to work fine.
Will unskip it